_posts/2019-10-29-sacred-geometry.md

```diff
@@ -0,0 +1,428 @@
+---
+title:  "Javascript Animation"
+date: 2019-10-29 00:00:00 +0800
+author: jimmy
+---
+
+<div style="width: 85px; float: left; margin-right: 20px;">
+    <img src="/assets/images/articles/tt_small.svg">
+</div>
+
+A quick look at drawing and animation with HTML Canvas and Javascript
+<!--more-->
+
+**TL;DR** Check out this cool javascript animation of multiplication tables <a href='#final_canvas'>below.</a>
+
+My very first experience with programming was on Khan Academy, going through their Drawing with Java tutorial. I hated
```

Comment:
maybe link to that tutorial?

Reply:
It doesn't exist anymore, at least not as it was at the time.